deployments/single/docker-compose.yml

```diff
@@ -49,7 +49,7 @@ services:
     environment:
       - VECTOR_STORE_PROVIDER=qdrant
       - QDRANT_URL=http://qdrant:6333
-      - QDRANT_COLLECTION=default
+      - QDRANT_COLLECTIONS=default
```

Comment:
isn't that just one app, shouldn't it only have one collection, hence keep `QDRANT_COLLECTION`?